content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -48,23 +50,25 @@ period a patch release may be issued from the release branch if a severe bug or
 security issue is found. Here are the latest Kubernetes releases and the support
 timeframe; which also applies to `kubeadm`.
 
+{{< table caption="Release and end-of-life details for recent Kubernetes versions" >}}
 | Kubernetes version | Release month  | End-of-life-month |
 |--------------------|----------------|-------------------|
-| v1.13.x            | December 2018  | September 2019    |
-| v1.14.x            | March 2019     | December 2019     |
+| v1.13.x            | December 2018  | September 2019    |
+| v1.14.x            | March 2019     | December 2019     |
 | v1.15.x            | June 2019      | March 2020        |
 | v1.16.x            | September 2019 | June 2020         |
+| v1.17.x            | December 2019  | September 2020    |
```

Comment:
i think it is time to remove this table and link to:
https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions